src/app/store/index.ts

```diff
@@ -86,6 +88,8 @@ export function createAppStore({
 	const communities = ref<Community[]>([]);
 	const communityStates = ref(new CommunityStates());
 
+	const firesides = ref<StoredFireside[]>([]);
```

Comment:
While we add/remove/update things in this list, it doesn't actually seem to be used for anything. Should we remove this and the related functions, or was there something else planned for this?

Reply:
Ah yeah, I wanted to implement a global Firesides channel that one can subscribe to for fireside expiration updates. I reverted those changes in this branch.